### PR TITLE
[JENKINS-39935] ManagedCredentialsBinding#addCredentials is not working

### PR DESCRIPTION
**Pull request by raul-arabaolaza**
[JENKINS-39935](https://issues.jenkins-ci.org/browse/JENKINS-39935)

@reviewbybees

### CODE REVIEW

**Comment by armfergom** [comment]
🐝

**Comment by varyvol** [comment]
🐝

File: src/test/java/plugins/CredentialsBindingTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        ManagedCredentialsBinding mcb = new ManagedCredentialsBinding(job);
+        SecretStringCredentialsBinding cb = mcb.addCredentialBinding(SecretStringCredentialsBinding.class);
+        cb.variable.set("BINDED_SECRET");
+        ShellBuildStep shell = job.addBuildStep(ShellBuildStep.class);
+        shell.command("if [ \"$BINDED_SECRET\" = \"" + SECRET_TEXT + "\" ] \n then \n echo \"true\" \n fi");
+        job.save();
+
+        Build b = job.scheduleBuild();
+        b.shouldSucceed();
+        assertThat(b.getConsole(), containsString("true"));
```
Comment (raul-arabaolaza) [inline src/test/java/plugins/CredentialsBindingTest.java]: You are right :)

**Comment by armfergom** [comment]
🐝

**Comment by varyvol** [comment]
🐝

**Comment by raul-arabaolaza** [comment]
@reviewbybees done

**Pull request merged**